### DISCUSSION TITLE
Before sharing a trained model can I have some help please?

### DISCUSSION THREAD

**Discussion by githubUser01946** [TheLastBen/fast-stable-diffusion#447] [Q&A]
![image](https://user-images.githubusercontent.com/117453319/200944057-83764270-3742-4a36-a5e0-b3fb24f09466.png)

Questions:
**1.** Is name_of_your_concept what shows **after** the sd-dreambooth-library/ part on the library?
**2.** Save_concept_to: My_Profile - If I choose this where does it go?
Its obvious where Public goes.
**3.** The huggingface token I used early on in the training is already set to write, can I use that on the form there.

**Comment by githubUser01946** [comment]
Furthermore when resuming the training its like it restarts the training and not actually just adding the 500 steps.
It started at 3000 steps and I didnt like the result so I changed it to 3500 and ticked the resume training box. Now its on step 750/3500 instead of x amount of the extra 500 like I thought it would be.

**Reply by TheLastBen** [reply]
The name of the concept is the main title of your model that shows in the library or your profile.
if you choose my_profile, you will find the model privately saved in your huggingface account.
if your huggingface toke is set to write, you can use the same token in that cell.

**Reply by TheLastBen** [reply]
when you resume training, you only specify how many steps you want to further train your model, if you want to train another 500 steps, you specify 500
